src/components/SideNavLink.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const SideNavLink = ({ image, name, linkTo }) => {
+  return (
+    <div className="flex justify-center items-center hover:bg-gray-200 py-1">
+      <img src={image} alt="icon" className="w-6" />
+      <Link
```

Comment:
As these links are on the sidebar. To have the ability of highlighting the active link. Use the Navlink instead of Link.

This component give you navData which can be used to set active state of the whole link element like the following example:
```
<NavLink className={(navData) =>navData.isActive
 ? 'py-1 px-2 font-bold my-1  cursor-pointer w-full bg-active'
: 'py-1 px-2 font-bold my-1  cursor-pointer w-full'}
 to="management" >
 <li>Management</li>
</NavLink>
```